lib/index.js

```diff
@@ -296,48 +340,24 @@ const Popover = createClass({
     this.untrackPopover()
   },
   animateExitStop () {
-    clearTimeout(this.exitingAnimationTimer1)
-    clearTimeout(this.exitingAnimationTimer2)
+    this.props.animateExitStop.call(this)
```

Comment:
should these handlers pass in refs to `this.containerEl` and/or `bodyEl` + `tipEl`, instead of relying on `this` binding?